Formula/traefik@1.rb

```diff
@@ -19,6 +19,9 @@ class TraefikAT1 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "69adbe00d348f14f732bbd9fe57d57e6f513f6f1c72f5ff2f4e09fae5b4ee2d9"
   end
 
+  # support ended 2021-12-31: https://doc.traefik.io/traefik/deprecation/releases/
+  deprecate! date: "2022-12-31", because: :unsupported
```

Comment:
```suggestion
  deprecate! date: "2022-08-03", because: :unsupported
```

Reply:
Or:
```suggestion
  disable! date: "2022-12-31", because: :unsupported
```

Reply:
Actually meant that to be
```suggestion
  disable! date: "2022-12-31", because: :unsupported
```